custom-war-packager-lib/src/main/java/io/jenkins/tools/warpackager/lib/util/DockerfileBuilder.java

```diff
@@ -57,7 +57,24 @@ public void build() throws IOException, InterruptedException {
             throw new IOException("Cannot build Docker image, tag is not defined");
         }
         LOGGER.log(Level.INFO, "Building Docker image {0}", tag);
-        SystemCommandHelper.processFor(outputDir, "docker", "build", "-t", tag, ".");
+        if (dockerSettings.isBuildx()) {
+            String output;
+            switch (dockerSettings.getOutput()) {
+                case "push":
+                    output = "--push";
+                    break;
+                case "load":
+                    output = "--load";
+                    break;
+                default:
```

Comment:
Fail on null and default?

Reply:
I add some comments in the code lines. The default behaviour is correct.